docs/LearnCalico.md

```diff
@@ -0,0 +1,364 @@
+# Calico networking in a containerized environment
+
+This GitHub repository is the entry point for running a Calico network in a
+containerized environment.
+
```

Comment:
Part of this ties in with where this doc/docs live.  It may be that we should beef up our initial README and then link to this for detailed info.